### PR TITLE
Translation:Guiding Participants,why_do_people_participate_in_open_source_communities

### PR DESCRIPTION
**Pull request by WUTONK**
修复了之前的错误pr

### CODE REVIEW

**Review by Cubik65536 — APPROVED** [review]
感觉还行， @LLLgoyour 来看看

File: translation/Guiding Participants/guiding_participants.md [hunk 1]
```diff
@@ -1,36 +1,27 @@
...
-- You want to understand what attracts people to participate in a pathway toward contribution.
-- You are looking to give your participation community a better experience.
-- You wish to build bridges to enable participants to become contributors.
-- You want to explore the benefit of user feedback in open source software.
-- You have reached the middle of this guidebook by starting at the beginning, and are working
-    your way toward the end.
+## 指导参与者们
+
+
+"参与"一个开源项目是一个有趣的、有点抽象的描述概念。”我只是一个用户！不用跟我谈论更多“和”我应该怎么样才能为这个项目作出贡献“之间的分界线就很明了。某个人对项目的参与从他与软件的关系已经不仅仅是使用它时开始。而且由于开源软件本质上是包容性和参与性的，因此大多数**用户**也变成了某种**参与者**，通常通过在项目的博客上留下评论、在项目的反馈跟踪器中提交问题或在社交媒体上参与项目。通过使用开源软件，他们**已经**开始参与让软件变得更好的过程。
```
Comment (KevinZonda) [inline translation/Guiding Participants/guiding_participants.md:4]: Use Chinese symbols.
```suggestion
“参与”一个开源项目是一个有趣的、有点抽象的描述概念。“我只是一个用户！不用跟我谈论更多”和”我应该怎么样才能为这个项目作出贡献“之间的分界线就很明了。某个人对项目的参与从他与软件的关系已经不仅仅是使用它时开始。而且由于开源软件本质上是包容性和参与性的，因此大多数**用户**也变成了某种**参与者**，通常通过在项目的博客上留下评论、在项目的反馈跟踪器中提交问题或在社交媒体上参与项目。通过使用开源软件，他们**已经**开始参与让软件变得更好的过程。
```

in `开源软件本质上是包容性和参与性的`, it's grammar structure should be SVP instead SP.

`通常通过在项目的博客上留下评论` lacks subject.

File: translation/Guiding Participants/guiding_participants.md [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
+
+以下是各种参与类型的一些示例：
+
+- 告诉朋友们你有多喜欢这个开源项目的软件或者目标
+- 在会议或者活动中，参与关于该软件的讨论
+- 在各种论坛上提出，赞成或解答有关与软件的问题。
+- 在笔记本电脑上贴上贴纸以示你对该软件的支持。
+- 就软件或项目向贡献者提供建设性反馈。
+- 推荐并提供帮助他人采用和使用本软件。
```
Comment (KevinZonda) [inline translation/Guiding Participants/guiding_participants.md:13]: Why some of them hv period symbols, some hv not?

File: translation/Guiding Participants/guiding_participants.md [hunk 3]
```diff
@@ -1,36 +1,27 @@
...
+
+本节包含以下章节：
+
+- 讨论参与动机背后的想法和动力。
+- 提供有关参与者如何使项目受益的详细信息。
+
+在以下情况下本节很适用：
```
Comment (KevinZonda) [inline translation/Guiding Participants/guiding_participants.md:20]: It looks weird.

File: translation/Guiding Participants/guiding_participants.md [hunk 4]
```diff
@@ -1,36 +1,27 @@
...
+
+- 你想了解由参与度较低的用户、热情的参与者或积极的贡献者组成的社区之间的差异
+- 你想了解是什么吸引人们参与到向软件贡献的途径中来。
+- 你希望为您的参与社区提供更好的体验。
+- 你希望建立一个桥梁来使参与者成为贡献者。
```
Comment (KevinZonda) [inline translation/Guiding Participants/guiding_participants.md:25]:
```suggestion
- 你希望建立一个来使参与者成为贡献者的桥梁。
```

File: translation/Guiding Participants/guiding_participants.md [hunk 5]
```diff
@@ -1,36 +1,27 @@
...
+本节包含以下章节：
+
+- 讨论参与动机背后的想法和动力。
+- 提供有关参与者如何使项目受益的详细信息。
+
+在以下情况下本节很适用：
+
+- 你想了解由参与度较低的用户、热情的参与者或积极的贡献者组成的社区之间的差异
+- 你想了解是什么吸引人们参与到向软件贡献的途径中来。
+- 你希望为您的参与社区提供更好的体验。
```
Comment (KevinZonda) [inline translation/Guiding Participants/guiding_participants.md:24]:
```suggestion
- 你希望为您参与的社区提供更好的体验。
```

File: translation/Guiding Participants/guiding_participants.md [hunk 6]
```diff
@@ -1,36 +1,27 @@
...
+- 推荐并提供帮助他人采用和使用本软件。
+
+本节包含以下章节：
+
+- 讨论参与动机背后的想法和动力。
+- 提供有关参与者如何使项目受益的详细信息。
+
+在以下情况下本节很适用：
+
+- 你想了解由参与度较低的用户、热情的参与者或积极的贡献者组成的社区之间的差异
```
Comment (KevinZonda) [inline translation/Guiding Participants/guiding_participants.md:22]: Why some of them hv period symbols, some hv not?

File: translation/Guiding Participants/why_do_people_participate_in_open_source_communities.md [hunk 7]
```diff
@@ -1,136 +1,61 @@
...
+
+**充分的内在激励因素，如学习和同伴认可**
+
+虽然不能取代更直接的好处，但获得同行认可，和在新技术领域工作的机会是也是一种激励因素。组织应该考虑到同行的认可，并明确地鼓励学习，以便最好地利用这些激励因素。
+
+**当激励因素过度轮换时可能适得其反** 
+
+正因为动机是来自多方面的，所以不要在任何一种动机上下太大的赌注。我们已经讨论了意识形态和利他主义的局限性。不过，也可以考虑一些“挠自己的痒处（scratch your own itch）”之类的东西。
+
+如果这是一个人提供贡献的唯一原因，他们会因为自己的具体需要而在项目中留下。这可能是一个完美的结果，但它不是一条发展长期维护者的道路。
```
Comment (KevinZonda) [inline translation/Guiding Participants/why_do_people_participate_in_open_source_communities.md:61]: Please separate.

**Review by KevinZonda — CHANGES_REQUESTED** [review]

**Comment by KevinZonda** [comment]
Thank you for your patient. Apologise to slow CR. Currently I am busier than usual and under extremely high pressure, it may take longer to response you. Apologise again for the inconvenience I caused. :)

**Comment by KevinZonda** [comment]
Also, currently, I have no Chinese IME, which means I cannot provide complete Chinese suggestions, but I'll gv a correct direction which you can fix everything up. In other words, you hv to read every review carefully so that you won't miss anything. I hope I can review your corrected content soon. Wish you every success with your future contribution. :)

**Review by LLLgoyour — CHANGES_REQUESTED** [review]

**Review by LLLgoyour — APPROVED** [review]

**Comment by Cubik65536** [comment]
Thanks @LLLgoyour @WUTONK

**Pull request merged**